python/py-pythonz/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        berdario pythonz 1.11.4 pythonz-
+name                py-${github.project}
+
+github.tarball_from tags
+
+platforms           darwin
+license             MIT
+
+python.versions     35 36 37
+
+maintainers         {gmail.com:rubendibattista @rubendibattista} openmaintainer
+description         Python installation manager supporting CPython, Stackless, PyPy and Jython
+long_description    $description
+checksums           rmd160  2be09cdf315e01bd8ff1f418243d5becec498a47 \
+                    sha256  6827909dfd4d52f5511bb2fe046d397d83e76812e9234228a84784d07ddf4cdc \
+                    size    84023
+
+if {${subport} ne ${name}} {
+    depends_lib     port:py${python.version}-setuptools
+    depends_lib     port:py${python.version}-resumable-urlretrieve
```

Comment:
you should use ```depends_lib-append``` as not to overwrite entries form the portgroup. Additionally, these two lines should be combined (right now you overwrite the ```py-setuptools``` dependency with the second line). Comment applies to all ports in this PR.

Reply:
Sorry. This was an overlook!